seaice_ecdr/intermediate_monthly.py

```diff
@@ -217,6 +217,18 @@ def get_daily_ds_for_month(
     }
 )
 
+CDR_SEAICE_CONC_MONTHLY_QA_FLAG_BITMASKS_SOUTH = OrderedDict(
+    {
+        "average_concentration_exceeds_0.15": 1,
+        "average_concentration_exceeds_0.30": 2,
+        "at_least_half_the_days_have_sea_ice_conc_exceeds_0.15": 4,
+        "at_least_half_the_days_have_sea_ice_conc_exceeds_0.30": 8,
+        "invalid_ice_mask_applied": 16,
+        "at_least_one_day_during_month_has_spatial_interpolation": 32,
+        "at_least_one_day_during_month_has_temporal_interpolation": 64,
+    }
+)
+
```

Comment:
```suggestion
CDR_SEAICE_CONC_MONTHLY_QA_FLAG_BITMASKS_SOUTH = CDR_SEAICE_CONC_MONTHLY_QA_FLAG_BITMASKS_NORTH.copy()
CDR_SEAICE_CONC_MONTHLY_QA_FLAG_BITMASKS_SOUTH.pop("at_least_one_day_during_month_has_melt_detected")

```

This makes it a little more clear that the SH bitmasks are the same as NH, just excluding the melt onset detection bitmask.